### ISSUE TITLE
project crashed

### ISSUE THREAD

**Issue by liu930410** [FernandezGFG/RobotArm-Simulation#3]
I cloned your project and try to complete it as debug mode, but when i run the project, nothing happened(no window shows) and it crashed. and if i delete view->setRootEntity(scene); ,it will show the window. something wrong happend ,could you help me check it,thanks

**Comment by gonzafernan** [comment]
Of course, I'll clone it again and see what the problem could be. Thanks for opening the issue!

**Comment by liu930410** [comment]
may i know what is your qt version and you use MSVC or MinGW to configure project
